### PR TITLE
fix: table layout shift while changing tabs

### PR DESCRIPTION
**Pull request by plxity**
fixes: #875 

https://github.com/documenso/documenso/assets/28510494/083fd87a-ef62-40e6-9696-9c04b4411502

### CODE REVIEW

**Comment by plxity** [comment]
@dguyen Please review this PR

**Comment by dguyen** [comment]
Please don't ping maintainers directly unless the PR is stale, we actively monitor the issues and PRs and will get around to them when possible

**Comment by plxity** [comment]
Noted.

File: packages/ui/primitives/data-table.tsx [hunk 1]
```diff
@@ -115,7 +115,12 @@ export function DataTable<TData, TValue>({
               table.getRowModel().rows.map((row) => (
                 <TableRow key={row.id} data-state={row.getIsSelected() && 'selected'}>
                   {row.getVisibleCells().map((cell) => (
-                    <TableCell key={cell.id}>
+                    <TableCell
+                      key={cell.id}
+                      style={{
+                        width: cell.column.getSize(),
+                      }}
```
Comment (Mythie) [inline packages/ui/primitives/data-table.tsx:122]: I would have to assume that this is covered by specifying the size already, is that not the case?

Comment (plxity) [reply packages/ui/primitives/data-table.tsx:122]: I didn't get your question, can you elaborate?

Comment (Mythie) [reply packages/ui/primitives/data-table.tsx:122]: I assume that tanstack table is already setting the width of the column using `cell.column.getSize()` so doing so in the style attribute here should be redundant.

Instead we should only need the manually specified size change that you had in the other part of this change.

Comment (plxity) [reply packages/ui/primitives/data-table.tsx:122]: Okay, So are you suggesting to hardcode the width in this file?

**Comment by plxity** [comment]
@Mythie Can you review this one?

**Review by Mythie — APPROVED** [review]

**Comment by Mythie** [comment]
Cheers Apoorv! 😄

**Pull request merged**